### DISCUSSION TITLE
(long) Reload the PDF after each highlight

### DISCUSSION THREAD

**Discussion by rouvenjahnke** [RyotaUshio/obsidian-pdf-plus#227] [Q&A]
Many thanks for the excellent plugin!
I have been looking for such a possibility for a long time and PDF++ is very helpful!
I use the plugin so far to make highlights directly in the PDF document. Unfortunately, every time I highlight text, the PDF reloads (presumably because the PDF has just been changed):

https://github.com/RyotaUshio/obsidian-pdf-plus/assets/141525719/4db94174-5c2f-4aaf-9b11-266587dec5ee

Is there perhaps a way around this?

**Comment by RyotaUshio** [comment]
Sorry for the late response. 
You might want to try this new plugin: https://github.com/Quorafind/Obsidian-PDF-Annotator

My note here: https://github.com/RyotaUshio/obsidian-pdf-plus/blob/main/README.md#note-on-saving-annotations-directly-in-pdf

**Reply by rouvenjahnke** [reply]
Thank you very much for your answer!

I'll Look at this plugin, the combination with your plugik seems like the best Option!
Thank you for your work!
